docs/getting_started/mod_state/index.md

```diff
@@ -1,64 +1,50 @@
 
 ## Using the CombatStateMachine
 
-### Adding situations
+For this demonstration we will be using the `FrayCombatStateMachine` however all the information regarding building the states and transitions apply to the general purpose `FraySimpleStateMachine` as well. To get started first add a `FrayCombatStateMachine` from the node creation dialog.
 
-To add a new situation call the `add_situation()` method on the combat state machine. It accepts 2 arguments, a string name, and a `GraphNodeStateMachineGlobal`. 
+Once the state machine is added to the scene you can begin adding situations. A situation is a name to `FrayRootState` mapping which represents the set of actions available to a combatant. To add a new situation call the `add_situation()` method on the combat state machine. It accepts 2 arguments, a string name, and a `FrayRootState`. The root state is what actually contains all the information related to states and transitions.
 
-Using the the situation builder the graph node can be built inline. Most builder methods return a reference to the builder allowing for chain method calls.
+Using the the root state builder the state machine can be assembled inline like so:
 
 ```gdscript
-var builder := Fray.State.CombatSituationBuilder.new()
-combat_state_machine.add_situation("on_ground", builder\
-    .transition_button("idle", "attack_1", "btn_punch")\
-    .transition_sequence("idle", "attack_2", "seq_236p")\
-    .build("idle")
-)
-```
 
-- The `transition_button()` and `transition_sequence()` method both take 4 arguments. A 'from' state string, a 'to' state string, a string representing the input name, and an optional config dictionary.
+@onready var combat_state_machine = $CombatStateMachine
+
+func _ready() -> void:
+    combat_state_machine.add_situation("on_ground", FrayRootState.builder()
+        .transition_button("idle", "attack_1", {input="btn_punch"})
+        .transition_sequence("idle", "attack_2", {sequence="seq_236p"})
+        .build("idle")
+    )
+```
 
-- The `build()` method will return a graph node built using the current build configuration. The first state added to the system will be used for the start state but you can optionally pass the start state to this method.
+- The `transition_button()` and `transition_sequence()` method both take 4 arguments. A 'from' state string, a 'to' state string, a string representing the input name, and an optional config dictionary. Note the input name is an arbitrary string name and does not inherently reference anything related to the input module.
```

Comment:
``The `transition_button()` and `transition_sequence()` method both take 3 arguments. A 'from' `StringName`, a 'to' `StringName`, and an optional config `Dictionary`.``*